.mockery.yaml

```diff
@@ -0,0 +1,11 @@
+with-expecter: false
+disable-version-string: true
+dir: internal/testutil/mocksvc
```

Comment:
[nit] since your interfaces are not centralized don't centralize the mocks, it leads to a lot of hooks if an interface happens to share a name and also to a lot of issues later on when you refactor your interfaces

Reply:
just be vigilant this doesn't become the permanent state, I'm currently trying to de-tangle some mock mess in the CLI and collocation would've made this easier